drape_frontend/route_renderer.cpp

```diff
@@ -383,21 +385,44 @@ CirclesPackHandle * RouteRenderer::GetPreviewHandle(size_t & index)
   return nullptr;
 }
 
-dp::Color RouteRenderer::GetMaskColor(RouteType routeType, double baseDistance,
-                                      bool arrows) const
+// Arrow mask is used to adjust route color for subroutes after the first stop. So subroute to the first stop
+// point has default color, while subsequent subroutes color is adjusted with RouteArrowsMaskCar,
+// RouteArrowsMaskBicycle and kRouteMaskPedestrian properties.
```

Comment:
This comment should be for GetArrowMaskColor ?

Reply:
Updated comment. Do you have ideas how to make it more clear?